File: Python Programming/problems on string manipulation/problem110.py
'''

110.Write a program which accept string from user reverse that string in place. 
	
	Input :   "abcd" 
	Output :   "dcba" 
	Input :   "abba" 
	Output :   "abba"

	Author : Milind Nalawade
	
	Date : 16/07/2021
	
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def Reverse(str):
	print("\n Output : {}  ".format("".join(reversed(str))))
#=============
#Entry point function
#=============	
def main():
	str=input("\n Enter your character : ")
	Reverse(str)
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: Java Programming/problems on string manipulation/problem110.java
/*

110.Write a program which accept string from user reverse that string in place. 
	
	Input :   "abcd" 
	Output :   "dcba" 
	Input :   "abba" 
	Output :   "abba"

	Author : Milind Nalawade
	
	Date : 16/07/2021
	
	Language : Java 14.0.2
	
*/
import java.lang.*;
import java .util.*;

class Logic
{
	public static void reverse(String str)
	{
		StringBuffer sbr = new StringBuffer(str);
        sbr.reverse();
		System.out.print("\n Output : "+sbr);
	}
}
class problem110 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print("\n Enter your string : ");
		String str=sobj.nextLine();
		reverse(str);
	}	
}